src/main/java/org/example/validators/SalesEmployeeValidator.java

```diff
@@ -0,0 +1,29 @@
+package org.example.validators;
+
+import org.example.exceptions.Entity;
+import org.example.exceptions.InvalidException;
+import org.example.models.SalesEmployeeRequest;
+
+public class SalesEmployeeValidator {
+    private final int nameLength = 50;
+    private final int bankAccNILength = 8;
```

Comment:
Please change this name as it was confusing to read that variable name in the code